src/Spotify/Playlist.java

```diff
@@ -0,0 +1,83 @@
+package Spotify;
+import java.io.*;
+import java.util.ArrayList;
+import Spotify.Library;
+import Spotify.Album;
+import Spotify.Playlist;
+
+public class Playlist extends Library {
+
+  // instance variable
+  private ArrayList <Song> Playlist = new ArrayList<Song>();
```

Comment:
`Playlist` is confusing as an instance variable name since it is the same name as the class and starts with an upper case letter (it shouldn't)